scripts/Modules/mkw_core.py

```diff
@@ -0,0 +1,98 @@
+from dolphin import memory
+from dataclasses import dataclass
+import math
+import mkw_classes as classes
+import mkw_translations as translate
+
+# NOTE (xi): wait for get_game_id() to be put in dolphin.memory before clearing
+#  these commented-out lines:
+
+# class RegionError(Exception):
+#    def __init__(self, message=f"Expected Mario Kart Wii game ID (RMCX01), got {get_game_id()}"):
+#        super().__init__(message)
```

Comment:
```suggestion
"""
class RegionError(Exception):
    def __init__(self, message=f"Expected Mario Kart Wii game ID (RMCX01), got {get_game_id()}"):
        super().__init__(message)
"""
```
Despite GitHub's syntax highlighting, this should be fine for a multi-line comment.